chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -633,6 +633,26 @@ std::vector<Array> DSplit(const Array& ary, std::vector<int64_t> indices) {
     return Split(ary, std::move(indices), 2);
 }
 
+std::vector<Array> VSplit(const Array& ary, int64_t sections) {
+    if (sections < 1) {
+        throw DimensionError("Number of sections must be larger than 0.");
+    }
+
+    if (ary.ndim() < 2) {
+        throw DimensionError("vsplit only works on arrays of 2 or more dimensions.");
```

Comment:
vsplit should work with 1-dimensional arrays also (since it's split over axis 0). Could you do that?

Reply:
The numpy implementation of [vsplit](https://github.com/numpy/numpy/blob/v1.16.1/numpy/lib/shape_base.py#L922-L972
) says it works with tow or more than two dimensions.

Reply:
Sorry you're right. Your current implementation looks correct.